db/seeds/available_provider_pairings.rb

```diff
@@ -0,0 +1,39 @@
+def describe_available_provider_pairing(app)
+  alp = app.active_lead_provider
+  print_seed_info("#{app.delivery_partner.name} are working with #{alp.lead_provider.name} in #{alp.registration_period.year}")
+end
+
+ambitious_institute = LeadProvider.find_by!(name: 'Ambitious Institute')
+teach_fast = LeadProvider.find_by!(name: 'Teach Fast')
+better_practice_network = LeadProvider.find_by!(name: 'Better Practice Network')
+
+active_lead_providers = ActiveLeadProvider
+  .eager_load(:registration_period, :lead_provider)
+  .index_by { |alp| [alp.lead_provider, alp.registration_period.year] }
+
+ambitious_institute_2021 = active_lead_providers.fetch([ambitious_institute, 2022])
+ambitious_institute_2022 = active_lead_providers.fetch([ambitious_institute, 2023])
+ambitious_institute_2023 = active_lead_providers.fetch([ambitious_institute, 2024])
```

Comment:
Shouldn't these names be ending in *_2022, *_2023 and *_2024 instead?

Reply:
Ah yeah good point. Will fix.